packages/demo-app-ts/src/Demo.css

```diff
@@ -23,6 +22,7 @@
 .pf-ri__topology-demo .pf-v6-c-toolbar {
   --pf-v6-c-toolbar__expandable-content--lg--PaddingBottom: 0;
   --pf-v6-c-toolbar--PaddingBottom: 0;
+  --pf-v6-c-toolbar--PaddingBlockStart: var(--pf-t--global--spacer--300);
```

Comment:
I see lots of references to base tokens in this CSS. Ideally we would use semantic tokens without `--300` or other numbers at the end.

Reply:
Thanks @nicolethoen ! Updated to use semantic tokens with the exception of `--pf-t--color--red--50` , which I couldn't find referenced with any semantic tokens in https://github.com/patternfly/patternfly/blob/v6/src/patternfly/base/tokens